server/src/controllers/organizations.controller.ts

```diff
@@ -80,7 +80,10 @@ export class OrganizationsController {
   @Post()
   async create(@Body('name') name, @User() user) {
     if (!name) {
-      throw new BadRequestException('name can not be empty');
+      throw new BadRequestException('name cannot be empty');
```

Comment:
Shall we create a DTO ([example](https://github.com/ToolJet/ToolJet/blob/develop/server/src/dto/version-create.dto.ts)) for this and add `@maxLength` validation

Reply:
Could that be a new issue? I'm running out of time for Hacktoberfest.